packages/okidoc-md/src/utils/nodeAST.js

```diff
@@ -21,6 +22,39 @@ function cleanUpFunction(node, { identifierName, JSDocCommentValue } = {}) {
   node.decorators = [];
   node.body = t.blockStatement([]);
 
+  if (node.typeParameters) {
+    /*
+      Removes generic constraint, because they can't be parsed by documentation.js
+      `function<T extends string>(param: T): T`
+     */
+    node.typeParameters.params = node.typeParameters.params.map(typeParam => {
+      typeParam.constraint = undefined;
+
+      return typeParam;
+    });
```

Comment:
I think that `forEach` would be a better fit here than `map`

```suggestion
    node.typeParameters.params.forEach(typeParam => {
      typeParam.constraint = undefined;
    });
```